.envrc

```
@@ -0,0 +1 @@
+eval "$(lorri direnv)"
```

Please check if lorri is in path (`if command -v lorry >/dev/null`) and fallback to `use nix` otherwise.

---

```suggestion
If type -P lorri; then
  eval "$(lorri direnv)"
else
  nix-shell
fi
```

---

More like:

```suggestion
if has lorri; then
  eval "$(lorri direnv)"
else
  use nix
fi
```

`has()` is a direnv builtin.